red_queen/cli.py

```diff
@@ -0,0 +1,185 @@
+# ------------------------------------------------------------------------------
+# Part of Red Queen Project.  This file is distributed under the Apache 2.0.
+# See accompanying file /LICENSE for details.
+# ------------------------------------------------------------------------------
+"""Shebang is used to make this code an python executable"""
```

Comment:
You don't really need to say this as it's a standard convention on unix systems for interpreted programs.
Instead you should have a module docstring that describes the purpose of the python code in this file